client/src/modules/grades/grades.routes.js

```diff
@@ -0,0 +1,42 @@
+angular.module('bhima.routes')
+.config(['$stateProvider', function ($stateProvider) {
+  $stateProvider
+    .state('grades', {
+      url         : '/grades',
+      controller  : 'GradeManagementController as GradeCtrl',
+      templateUrl : 'modules/grades/grades.html',
+    })
+
+    .state('grades.create', {
+      url : '/create',
+      params : {
+        grade : { value : null },
+        creating : { value : true },
+      },
+      onEnter : ['$uibModal', gradeModal],
+      onExit : ['$uibModalStack', closeModal],
+    })
+
+    .state('grades.edit', {
+      url : '/:uuid/edit',
+      params : {
+        grade : { value : null },
```

Comment:
I think this is probably supposed to be `uuid` now.  But if it works without it, then it might not be necessary.